src/main/java/frc/robot/RobotContainer.java

```diff
@@ -80,10 +81,10 @@ public class RobotContainer {
   private final Elevator elevator = new Elevator();
   private final CoralScorer coralScorer = new CoralScorer();
   private final IntakeSubsystem intake;
+  private IntakeModeEnum intakeMode = IntakeModeEnum.GROUND_INTAKE;
+
+  private boolean elevatorL1 = true;
```

Comment:
Previous naming was more descriptive. elevatorNotL1 describes the mapping: 
true -> elevator
false -> L1